admin_guide/install/prerequisites.adoc

```diff
@@ -295,7 +323,7 @@ dm.thinpooldev=/dev/mapper/docker--vg-docker--pool
 +
 [WARNING]
 ====
-This will destroy any docker containers or images currently on the host.
+This will destroy any Docker containers or images currently on the host.
 ====
```

Comment:
in #680 I've added a section about reconfiguring things after having run docker-storage-setup. I haven't listed the exact commands as I think it's a bit risky so I've linked to the LVM admin guide.